frontend/actions/documentsActions.js

```diff
@@ -33,6 +33,41 @@ export function deleteDocuments ({api, collection, ids}) {
   }
 }
 
+export function deleteMedia ({api, collection, ids}) {
+  return (dispatch, getState) => {
+    Promise.all(
+      ids.map(id =>
+        fetch(
+          `${api.host}:${api.port}/media/${id}`,
+          {
+            headers: {
+              'Accept': 'application/json',
+              'Authorization': `Bearer ${getState().user.accessToken}`
+            },
+            method: 'DELETE'
+          }
+        )
+      )
+    )
+    .then(
+      responses => {
```

Comment:
Could you make the function argument part of the same line as the function name to avoid an unnecessary level of indentation?

Reply:
This in general I shall try to stick to.